README.md

```diff
@@ -64,16 +58,4 @@ $ curl http://localhost:9410/api/v0/token -H 'content-type: application/json' -d
 }'
 ```
 
-## Configuration
-
-All configuration in the system is currently through command-line flags. The available flags for each
-component can be viewed using the `-h` flag. The configuration is mostly limited to connection details for the
-various components and their dependencies. As mentioned in [Getting started](#getting-started) the allowed charge
-stations and tokens are currently hard-coded in the server start up commands.
-
-## Contributing
-
-Learn more about how to contribute on this project through [Contributing](./CONTRIBUTING.md)
-
-## License
```

Comment:
We want to highlight the license.

Reply:
most people know about LICENSE file and it's highlighted by github